cpu-bind=MASK - bp1-gpu020, task  0  0 [1725629]: mask 0x101 set
Error executing job with overrides: ['seed=0', 'experiment=bp_experiment', 'model=layers/gnn', 'model@critic_model=layers/deepsets', 'algorithm=maddpg', 'task=vmas/painting']
Traceback (most recent call last):
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/probabilistic.py", line 338, in get_dist
    dist = self.distribution_class(**dist_kwargs, **self.distribution_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torchrl/modules/distributions/continuous.py", line 562, in __init__
    base = Delta(
           ^^^^^^
TypeError: Delta.__init__() got an unexpected keyword argument 'low'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/user/work/vd20433/BenchMARL/benchmarl/run.py", line 37, in hydra_experiment
    experiment = load_experiment_from_hydra(cfg, task_name=task_name)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/work/vd20433/BenchMARL/benchmarl/hydra_config.py", line 41, in load_experiment_from_hydra
    return Experiment(
           ^^^^^^^^^^^
  File "/user/work/vd20433/BenchMARL/benchmarl/experiment/experiment.py", line 336, in __init__
    self._setup()
  File "/user/work/vd20433/BenchMARL/benchmarl/experiment/experiment.py", line 358, in _setup
    self._setup_collector()
  File "/user/work/vd20433/BenchMARL/benchmarl/experiment/experiment.py", line 481, in _setup_collector
    self.collector = SyncDataCollector(
                     ^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torchrl/collectors/collectors.py", line 791, in __init__
    policy_output = self.policy(policy_input)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 601, in new_fun
    raise err
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 589, in new_fun
    return getattr(type(self), fun_name)(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/common.py", line 291, in wrapper
    return func(_self, tensordict, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/_contextlib.py", line 126, in decorate_context
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/utils.py", line 261, in wrapper
    return func(_self, tensordict, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/sequence.py", line 428, in forward
    tensordict = self._run_module(module, tensordict, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/sequence.py", line 409, in _run_module
    tensordict = module(tensordict, **kwargs)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1561, in _call_impl
    result = forward_call(*args, **kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 601, in new_fun
    raise err
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 589, in new_fun
    return getattr(type(self), fun_name)(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torchrl/modules/tensordict_module/exploration.py", line 380, in forward
    tensordict = self.td_module.forward(tensordict)
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 601, in new_fun
    raise err
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 589, in new_fun
    return getattr(type(self), fun_name)(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/common.py", line 291, in wrapper
    return func(_self, tensordict, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/_contextlib.py", line 126, in decorate_context
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/utils.py", line 261, in wrapper
    return func(_self, tensordict, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/probabilistic.py", line 555, in forward
    return self.module[-1](tensordict_out, _requires_sample=self._requires_sample)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1511, in _wrapped_call_impl
    return self._call_impl(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/torch/nn/modules/module.py", line 1520, in _call_impl
    return forward_call(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 601, in new_fun
    raise err
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/functional_modules.py", line 589, in new_fun
    return getattr(type(self), fun_name)(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/common.py", line 291, in wrapper
    return func(_self, tensordict, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/_contextlib.py", line 126, in decorate_context
    return func(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/utils.py", line 261, in wrapper
    return func(_self, tensordict, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/probabilistic.py", line 382, in forward
    dist = self.get_dist(tensordict)
           ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/user/home/vd20433/work/miniconda3/envs/benchmarl/lib/python3.11/site-packages/tensordict/nn/probabilistic.py", line 341, in get_dist
    raise TypeError(
TypeError: distribution keywords and tensordict keys indicated by ProbabilisticTensorDictModule.dist_keys must match.Got this error message: 
    Delta.__init__() got an unexpected keyword argument 'low'
with dist_keys=[('nav_agents', 'param')]

Set the environment variable HYDRA_FULL_ERROR=1 for a complete stack trace.
